UpSquared Grub build docs don't work on Ubuntu 20.04

**Describe the bug**
I'm following instructions on https://docs.zephyrproject.org/latest/boards/x86/up_squared/doc/index.html

On the step where I run:

```
$ ./boards/x86/common/scripts/build_grub.sh x86_64
```

I get the following output:

```
$ ./boards/x86/common/scripts/build_grub.sh x86_64
~/projects/zephyr/zephyr/boards/x86/common/scripts/grub ~/projects/zephyr/zephyr
Cloning into 'src'...
remote: Counting objects: 94771, done.
remote: Compressing objects: 100% (21811/21811), done.
remote: Total 94771 (delta 70661), reused 94694 (delta 70616)
Receiving objects: 100% (94771/94771), 70.39 MiB | 4.48 MiB/s, done.
Resolving deltas: 100% (70661/70661), done.
~/projects/zephyr/zephyr/boards/x86/common/scripts/grub/src ~/projects/zephyr/zephyr/boards/x86/common/scripts/grub ~/projects/zephyr/zephyr
Note: switching to 'grub-2.04-rc1-17-g8e8723a6b'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 8e8723a6b f2fs: Disable gcc9 -Waddress-of-packed-member
~/projects/zephyr/zephyr/boards/x86/common/scripts/grub ~/projects/zephyr/zephyr
~/projects/zephyr/zephyr/boards/x86/common/scripts/grub/src ~/projects/zephyr/zephyr/boards/x86/common/scripts/grub ~/projects/zephyr/zephyr
./bootstrap: Bootstrapping from checked-out grub sources...
./bootstrap: consider installing git-merge-changelog from gnulib
./bootstrap: getting gnulib files...
Cloning into 'gnulib'...
remote: Counting objects: 220896, done.
remote: Compressing objects: 100% (28337/28337), done.
remote: Total 220896 (delta 192700), reused 220605 (delta 192491)
Receiving objects: 100% (220896/220896), 45.28 MiB | 3.29 MiB/s, done.
Resolving deltas: 100% (192700/192700), done.
Note: switching to 'd271f868a8df9bbec29049d01e056481b7a1a263'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at d271f868a lock: Fix link error with --enable-threads=pth.
./bootstrap: autopoint --force
Copying file ABOUT-NLS
Copying file build-aux/config.rpath
Creating directory m4
Copying file m4/codeset.m4
Copying file m4/extern-inline.m4
Copying file m4/fcntl-o.m4
Copying file m4/gettext.m4
Copying file m4/glibc2.m4
Copying file m4/glibc21.m4
Copying file m4/iconv.m4
Copying file m4/intdiv0.m4
Copying file m4/intl.m4
Copying file m4/intldir.m4
Copying file m4/intlmacosx.m4
Copying file m4/intmax.m4
Copying file m4/inttypes-pri.m4
Copying file m4/inttypes_h.m4
Copying file m4/lcmessage.m4
Copying file m4/lib-ld.m4
Copying file m4/lib-link.m4
Copying file m4/lib-prefix.m4
Copying file m4/lock.m4
Copying file m4/longlong.m4
Copying file m4/nls.m4
Copying file m4/po.m4
Copying file m4/printf-posix.m4
Copying file m4/progtest.m4
Copying file m4/size_max.m4
Copying file m4/stdint_h.m4
Copying file m4/threadlib.m4
Copying file m4/uintmax_t.m4
Copying file m4/visibility.m4
Copying file m4/wchar_t.m4
Copying file m4/wint_t.m4
Copying file m4/xsize.m4
Copying file po/Makefile.in.in
Copying file po/Makevars.template
Copying file po/Rules-quot
./bootstrap: gnulib/gnulib-tool    --no-changelog   --aux-dir=build-aux   --doc-base=doc   --lib=libgnu   --m4-base=m4/   --source-base=grub-core/lib/gnulib/   --tests-base=tests   --local-dir=gl     --no-conditional-dependencies   --no-vc-files    --import ...
Module list with included dependencies (indented):
    absolute-header
    alloca
    alloca-opt
  argp
    assure
    at-internal
    btowc
    builtin-expect
    chdir
    chdir-long
    cloexec
    close
    dirent
    dirfd
    dirname-lgpl
    dosname
    double-slash-root
    dup2
    errno
  error
    exitfail
    extensions
    extern-inline
    fchdir
    fcntl
    fcntl-h
    fd-hook
    fd-safer-flag
    filename
    filenamecat-lgpl
    flexmember
    float
  fnmatch
    fnmatch-h
    fstat
    getcwd-lgpl
  getdelim
    getdtablesize
  getline
    getopt-gnu
    getopt-posix
    getprogname
  gettext-h
  gitlog-to-changelog
    hard-locale
    havelib
    include_next
    intprops
    langinfo
    largefile
    libc-config
    limits-h
    localcharset
    locale
    localeconv
    lock
    malloc-gnu
    malloc-posix
    malloca
    mbrtowc
    mbsinit
    mbsrtowcs
  mbswidth
    mbtowc
    memchr
    mempcpy
    memrchr
    msvc-inval
    msvc-nothrow
    multiarch
    nl_langinfo
    nocrash
    open
    openat
    openat-die
    openat-h
    pathmax
  progname
    rawmemchr
  realloc-gnu
    realloc-posix
  regex
  save-cwd
    size_max
    sleep
    snippet/_Noreturn
    snippet/arg-nonnull
    snippet/c++defs
    snippet/warn-on-use
    ssize_t
    stat
    stat-time
    stdalign
    stdbool
    stddef
    stdint
    stdio
    stdlib
    strcase
    strchrnul
    strdup-posix
    streq
    strerror
    strerror-override
    string
    strings
    strndup
    strnlen
    strnlen1
    sys_stat
    sys_types
    sysexits
    threadlib
    time
    unistd
    unistd-safer
    unitypes
    uniwidth/base
    uniwidth/width
    vasnprintf
    verify
    vsnprintf
    wchar
    wcrtomb
    wctype-h
    wcwidth
    xalloc-oversized
    xsize
Notice from module argp:
  If you are using GNU gettext version 0.16.1 or older, add the following options
  to XGETTEXT_OPTIONS in your po/Makevars:
    --flag=argp_error:2:c-format --flag=argp_failure:4:c-format
Notice from module error:
  If you are using GNU gettext version 0.16.1 or older, add the following options
  to XGETTEXT_OPTIONS in your po/Makevars:
    --flag=error:3:c-format --flag=error_at_line:5:c-format
File list:
  build-aux/config.rpath
  build-aux/gitlog-to-changelog
  lib/_Noreturn.h
  lib/alloca.c
  lib/alloca.in.h
  lib/arg-nonnull.h
  lib/argp-ba.c
  lib/argp-eexst.c
  lib/argp-fmtstream.c
  lib/argp-fmtstream.h
  lib/argp-fs-xinl.c
  lib/argp-help.c
  lib/argp-namefrob.h
  lib/argp-parse.c
  lib/argp-pin.c
  lib/argp-pv.c
  lib/argp-pvh.c
  lib/argp-xinl.c
  lib/argp.h
  lib/asnprintf.c
  lib/assure.h
  lib/basename-lgpl.c
  lib/btowc.c
  lib/c++defs.h
  lib/cdefs.h
  lib/chdir-long.c
  lib/chdir-long.h
  lib/cloexec.c
  lib/cloexec.h
  lib/close.c
  lib/dirent.in.h
  lib/dirfd.c
  lib/dirname-lgpl.c
  lib/dirname.h
  lib/dosname.h
  lib/dup-safer-flag.c
  lib/dup-safer.c
  lib/dup2.c
  lib/errno.in.h
  lib/error.c
  lib/error.h
  lib/exitfail.c
  lib/exitfail.h
  lib/fchdir.c
  lib/fcntl.c
  lib/fcntl.in.h
  lib/fd-hook.c
  lib/fd-hook.h
  lib/fd-safer-flag.c
  lib/fd-safer.c
  lib/filename.h
  lib/filenamecat-lgpl.c
  lib/filenamecat.h
  lib/flexmember.h
  lib/float+.h
  lib/float.c
  lib/float.in.h
  lib/fnmatch.c
  lib/fnmatch.in.h
  lib/fnmatch_loop.c
  lib/fstat.c
  lib/getcwd-lgpl.c
  lib/getdelim.c
  lib/getdtablesize.c
  lib/getline.c
  lib/getopt-cdefs.in.h
  lib/getopt-core.h
  lib/getopt-ext.h
  lib/getopt-pfx-core.h
  lib/getopt-pfx-ext.h
  lib/getopt.c
  lib/getopt.in.h
  lib/getopt1.c
  lib/getopt_int.h
  lib/getprogname.c
  lib/getprogname.h
  lib/gettext.h
  lib/glthread/lock.c
  lib/glthread/lock.h
  lib/glthread/threadlib.c
  lib/hard-locale.c
  lib/hard-locale.h
  lib/intprops.h
  lib/itold.c
  lib/langinfo.in.h
  lib/libc-config.h
  lib/limits.in.h
  lib/localcharset.c
  lib/localcharset.h
  lib/locale.in.h
  lib/localeconv.c
  lib/malloc.c
  lib/malloca.c
  lib/malloca.h
  lib/mbrtowc.c
  lib/mbsinit.c
  lib/mbsrtowcs-impl.h
  lib/mbsrtowcs-state.c
  lib/mbsrtowcs.c
  lib/mbswidth.c
  lib/mbswidth.h
  lib/mbtowc-impl.h
  lib/mbtowc.c
  lib/memchr.c
  lib/memchr.valgrind
  lib/mempcpy.c
  lib/memrchr.c
  lib/msvc-inval.c
  lib/msvc-inval.h
  lib/msvc-nothrow.c
  lib/msvc-nothrow.h
  lib/nl_langinfo.c
  lib/open.c
  lib/openat-die.c
  lib/openat-priv.h
  lib/openat-proc.c
  lib/openat.c
  lib/openat.h
  lib/pathmax.h
  lib/pipe-safer.c
  lib/printf-args.c
  lib/printf-args.h
  lib/printf-parse.c
  lib/printf-parse.h
  lib/progname.c
  lib/progname.h
  lib/rawmemchr.c
  lib/rawmemchr.valgrind
  lib/realloc.c
  lib/regcomp.c
  lib/regex.c
  lib/regex.h
  lib/regex_internal.c
  lib/regex_internal.h
  lib/regexec.c
  lib/save-cwd.c
  lib/save-cwd.h
  lib/size_max.h
  lib/sleep.c
  lib/stat-time.c
  lib/stat-time.h
  lib/stat-w32.c
  lib/stat-w32.h
  lib/stat.c
  lib/stdalign.in.h
  lib/stdbool.in.h
  lib/stddef.in.h
  lib/stdint.in.h
  lib/stdio.in.h
  lib/stdlib.in.h
  lib/strcasecmp.c
  lib/strchrnul.c
  lib/strchrnul.valgrind
  lib/strdup.c
  lib/streq.h
  lib/strerror-override.c
  lib/strerror-override.h
  lib/strerror.c
  lib/string.in.h
  lib/strings.in.h
  lib/stripslash.c
  lib/strncasecmp.c
  lib/strndup.c
  lib/strnlen.c
  lib/strnlen1.c
  lib/strnlen1.h
  lib/sys_stat.in.h
  lib/sys_types.in.h
  lib/sysexits.in.h
  lib/time.in.h
  lib/unistd--.h
  lib/unistd-safer.h
  lib/unistd.c
  lib/unistd.in.h
  lib/unitypes.in.h
  lib/uniwidth.in.h
  lib/uniwidth/cjk.h
  lib/uniwidth/width.c
  lib/vasnprintf.c
  lib/vasnprintf.h
  lib/verify.h
  lib/vsnprintf.c
  lib/warn-on-use.h
  lib/wchar.in.h
  lib/wcrtomb.c
  lib/wctype-h.c
  lib/wctype.in.h
  lib/wcwidth.c
  lib/xalloc-oversized.h
  lib/xsize.c
  lib/xsize.h
  m4/00gnulib.m4
  m4/__inline.m4
  m4/absolute-header.m4
  m4/alloca.m4
  m4/argp.m4
  m4/btowc.m4
  m4/builtin-expect.m4
  m4/chdir-long.m4
  m4/close.m4
  m4/codeset.m4
  m4/dirent_h.m4
  m4/dirfd.m4
  m4/dirname.m4
  m4/double-slash-root.m4
  m4/dup2.m4
  m4/eealloc.m4
  m4/errno_h.m4
  m4/error.m4
  m4/exponentd.m4
  m4/extensions.m4
  m4/extern-inline.m4
  m4/fchdir.m4
  m4/fcntl-o.m4
  m4/fcntl.m4
  m4/fcntl_h.m4
  m4/filenamecat.m4
  m4/flexmember.m4
  m4/float_h.m4
  m4/fnmatch.m4
  m4/fnmatch_h.m4
  m4/fstat.m4
  m4/getcwd.m4
  m4/getdelim.m4
  m4/getdtablesize.m4
  m4/getline.m4
  m4/getopt.m4
  m4/getprogname.m4
  m4/glibc21.m4
  m4/gnulib-common.m4
  m4/host-cpu-c-abi.m4
  m4/include_next.m4
  m4/intmax_t.m4
  m4/inttypes_h.m4
  m4/langinfo_h.m4
  m4/largefile.m4
  m4/lib-ld.m4
  m4/lib-link.m4
  m4/lib-prefix.m4
  m4/libunistring-base.m4
  m4/limits-h.m4
  m4/localcharset.m4
  m4/locale-fr.m4
  m4/locale-ja.m4
  m4/locale-zh.m4
  m4/locale_h.m4
  m4/localeconv.m4
  m4/lock.m4
  m4/longlong.m4
  m4/lstat.m4
  m4/malloc.m4
  m4/malloca.m4
  m4/math_h.m4
  m4/mbrtowc.m4
  m4/mbsinit.m4
  m4/mbsrtowcs.m4
  m4/mbstate_t.m4
  m4/mbswidth.m4
  m4/mbtowc.m4
  m4/memchr.m4
  m4/mempcpy.m4
  m4/memrchr.m4
  m4/mmap-anon.m4
  m4/mode_t.m4
  m4/msvc-inval.m4
  m4/msvc-nothrow.m4
  m4/multiarch.m4
  m4/nl_langinfo.m4
  m4/nocrash.m4
  m4/off_t.m4
  m4/open-cloexec.m4
  m4/open.m4
  m4/openat.m4
  m4/pathmax.m4
  m4/printf.m4
  m4/pthread_rwlock_rdlock.m4
  m4/rawmemchr.m4
  m4/realloc.m4
  m4/regex.m4
  m4/save-cwd.m4
  m4/size_max.m4
  m4/sleep.m4
  m4/ssize_t.m4
  m4/stat-time.m4
  m4/stat.m4
  m4/stdalign.m4
  m4/stdbool.m4
  m4/stddef_h.m4
  m4/stdint.m4
  m4/stdint_h.m4
  m4/stdio_h.m4
  m4/stdlib_h.m4
  m4/strcase.m4
  m4/strchrnul.m4
  m4/strdup.m4
  m4/strerror.m4
  m4/string_h.m4
  m4/strings_h.m4
  m4/strndup.m4
  m4/strnlen.m4
  m4/sys_socket_h.m4
  m4/sys_stat_h.m4
  m4/sys_types_h.m4
  m4/sysexits.m4
  m4/threadlib.m4
  m4/time_h.m4
  m4/unistd-safer.m4
  m4/unistd_h.m4
  m4/vasnprintf.m4
  m4/vsnprintf.m4
  m4/warn-on-use.m4
  m4/wchar_h.m4
  m4/wchar_t.m4
  m4/wcrtomb.m4
  m4/wctype_h.m4
  m4/wcwidth.m4
  m4/wint_t.m4
  m4/xsize.m4
Creating directory ./grub-core/lib/gnulib
Creating directory ./grub-core/lib/gnulib/glthread
Creating directory ./grub-core/lib/gnulib/uniwidth
Replacing file build-aux/config.rpath (non-gnulib code backed up in build-aux/config.rpath~) !!
Copying file build-aux/gitlog-to-changelog
Copying file grub-core/lib/gnulib/_Noreturn.h
Copying file grub-core/lib/gnulib/alloca.c
Copying file grub-core/lib/gnulib/alloca.in.h
Copying file grub-core/lib/gnulib/arg-nonnull.h
Copying file grub-core/lib/gnulib/argp-ba.c
Copying file grub-core/lib/gnulib/argp-eexst.c
Copying file grub-core/lib/gnulib/argp-fmtstream.c
Copying file grub-core/lib/gnulib/argp-fmtstream.h
Copying file grub-core/lib/gnulib/argp-fs-xinl.c
Copying file grub-core/lib/gnulib/argp-help.c
Copying file grub-core/lib/gnulib/argp-namefrob.h
Copying file grub-core/lib/gnulib/argp-parse.c
Copying file grub-core/lib/gnulib/argp-pin.c
Copying file grub-core/lib/gnulib/argp-pv.c
Copying file grub-core/lib/gnulib/argp-pvh.c
Copying file grub-core/lib/gnulib/argp-xinl.c
Copying file grub-core/lib/gnulib/argp.h
Copying file grub-core/lib/gnulib/asnprintf.c
Copying file grub-core/lib/gnulib/assure.h
Copying file grub-core/lib/gnulib/basename-lgpl.c
Copying file grub-core/lib/gnulib/btowc.c
Copying file grub-core/lib/gnulib/c++defs.h
Copying file grub-core/lib/gnulib/cdefs.h
Copying file grub-core/lib/gnulib/chdir-long.c
Copying file grub-core/lib/gnulib/chdir-long.h
Copying file grub-core/lib/gnulib/cloexec.c
Copying file grub-core/lib/gnulib/cloexec.h
Copying file grub-core/lib/gnulib/close.c
Copying file grub-core/lib/gnulib/dirent.in.h
Copying file grub-core/lib/gnulib/dirfd.c
Copying file grub-core/lib/gnulib/dirname-lgpl.c
Copying file grub-core/lib/gnulib/dirname.h
Copying file grub-core/lib/gnulib/dosname.h
Copying file grub-core/lib/gnulib/dup-safer-flag.c
Copying file grub-core/lib/gnulib/dup-safer.c
Copying file grub-core/lib/gnulib/dup2.c
Copying file grub-core/lib/gnulib/errno.in.h
Copying file grub-core/lib/gnulib/error.c
Copying file grub-core/lib/gnulib/error.h
Copying file grub-core/lib/gnulib/exitfail.c
Copying file grub-core/lib/gnulib/exitfail.h
Copying file grub-core/lib/gnulib/fchdir.c
Copying file grub-core/lib/gnulib/fcntl.c
Copying file grub-core/lib/gnulib/fcntl.in.h
Copying file grub-core/lib/gnulib/fd-hook.c
Copying file grub-core/lib/gnulib/fd-hook.h
Copying file grub-core/lib/gnulib/fd-safer-flag.c
Copying file grub-core/lib/gnulib/fd-safer.c
Copying file grub-core/lib/gnulib/filename.h
Copying file grub-core/lib/gnulib/filenamecat-lgpl.c
Copying file grub-core/lib/gnulib/filenamecat.h
Copying file grub-core/lib/gnulib/flexmember.h
Copying file grub-core/lib/gnulib/float+.h
Copying file grub-core/lib/gnulib/float.c
Copying file grub-core/lib/gnulib/float.in.h
Copying file grub-core/lib/gnulib/fnmatch.c
Copying file grub-core/lib/gnulib/fnmatch.in.h
Copying file grub-core/lib/gnulib/fnmatch_loop.c
Copying file grub-core/lib/gnulib/fstat.c
Copying file grub-core/lib/gnulib/getcwd-lgpl.c
Copying file grub-core/lib/gnulib/getdelim.c
Copying file grub-core/lib/gnulib/getdtablesize.c
Copying file grub-core/lib/gnulib/getline.c
Copying file grub-core/lib/gnulib/getopt-cdefs.in.h
Copying file grub-core/lib/gnulib/getopt-core.h
Copying file grub-core/lib/gnulib/getopt-ext.h
Copying file grub-core/lib/gnulib/getopt-pfx-core.h
Copying file grub-core/lib/gnulib/getopt-pfx-ext.h
Copying file grub-core/lib/gnulib/getopt.c
Copying file grub-core/lib/gnulib/getopt.in.h
Copying file grub-core/lib/gnulib/getopt1.c
Copying file grub-core/lib/gnulib/getopt_int.h
Copying file grub-core/lib/gnulib/getprogname.c
Copying file grub-core/lib/gnulib/getprogname.h
Copying file grub-core/lib/gnulib/gettext.h
Copying file grub-core/lib/gnulib/glthread/lock.c
Copying file grub-core/lib/gnulib/glthread/lock.h
Copying file grub-core/lib/gnulib/glthread/threadlib.c
Copying file grub-core/lib/gnulib/hard-locale.c
Copying file grub-core/lib/gnulib/hard-locale.h
Copying file grub-core/lib/gnulib/intprops.h
Copying file grub-core/lib/gnulib/itold.c
Copying file grub-core/lib/gnulib/langinfo.in.h
Copying file grub-core/lib/gnulib/libc-config.h
Copying file grub-core/lib/gnulib/limits.in.h
Copying file grub-core/lib/gnulib/localcharset.c
Copying file grub-core/lib/gnulib/localcharset.h
Copying file grub-core/lib/gnulib/locale.in.h
Copying file grub-core/lib/gnulib/localeconv.c
Copying file grub-core/lib/gnulib/malloc.c
Copying file grub-core/lib/gnulib/malloca.c
Copying file grub-core/lib/gnulib/malloca.h
Copying file grub-core/lib/gnulib/mbrtowc.c
Copying file grub-core/lib/gnulib/mbsinit.c
Copying file grub-core/lib/gnulib/mbsrtowcs-impl.h
Copying file grub-core/lib/gnulib/mbsrtowcs-state.c
Copying file grub-core/lib/gnulib/mbsrtowcs.c
Copying file grub-core/lib/gnulib/mbswidth.c
Copying file grub-core/lib/gnulib/mbswidth.h
Copying file grub-core/lib/gnulib/mbtowc-impl.h
Copying file grub-core/lib/gnulib/mbtowc.c
Copying file grub-core/lib/gnulib/memchr.c
Copying file grub-core/lib/gnulib/memchr.valgrind
Copying file grub-core/lib/gnulib/mempcpy.c
Copying file grub-core/lib/gnulib/memrchr.c
Copying file grub-core/lib/gnulib/msvc-inval.c
Copying file grub-core/lib/gnulib/msvc-inval.h
Copying file grub-core/lib/gnulib/msvc-nothrow.c
Copying file grub-core/lib/gnulib/msvc-nothrow.h
Copying file grub-core/lib/gnulib/nl_langinfo.c
Copying file grub-core/lib/gnulib/open.c
Copying file grub-core/lib/gnulib/openat-die.c
Copying file grub-core/lib/gnulib/openat-priv.h
Copying file grub-core/lib/gnulib/openat-proc.c
Copying file grub-core/lib/gnulib/openat.c
Copying file grub-core/lib/gnulib/openat.h
Copying file grub-core/lib/gnulib/pathmax.h
Copying file grub-core/lib/gnulib/pipe-safer.c
Copying file grub-core/lib/gnulib/printf-args.c
Copying file grub-core/lib/gnulib/printf-args.h
Copying file grub-core/lib/gnulib/printf-parse.c
Copying file grub-core/lib/gnulib/printf-parse.h
Copying file grub-core/lib/gnulib/progname.c
Copying file grub-core/lib/gnulib/progname.h
Copying file grub-core/lib/gnulib/rawmemchr.c
Copying file grub-core/lib/gnulib/rawmemchr.valgrind
Copying file grub-core/lib/gnulib/realloc.c
Copying file grub-core/lib/gnulib/regcomp.c
Copying file grub-core/lib/gnulib/regex.c
Copying file grub-core/lib/gnulib/regex.h
Copying file grub-core/lib/gnulib/regex_internal.c
Copying file grub-core/lib/gnulib/regex_internal.h
Copying file grub-core/lib/gnulib/regexec.c
Copying file grub-core/lib/gnulib/save-cwd.c
Copying file grub-core/lib/gnulib/save-cwd.h
Copying file grub-core/lib/gnulib/size_max.h
Copying file grub-core/lib/gnulib/sleep.c
Copying file grub-core/lib/gnulib/stat-time.c
Copying file grub-core/lib/gnulib/stat-time.h
Copying file grub-core/lib/gnulib/stat-w32.c
Copying file grub-core/lib/gnulib/stat-w32.h
Copying file grub-core/lib/gnulib/stat.c
Copying file grub-core/lib/gnulib/stdalign.in.h
Copying file grub-core/lib/gnulib/stdbool.in.h
Copying file grub-core/lib/gnulib/stddef.in.h
Copying file grub-core/lib/gnulib/stdint.in.h
Copying file grub-core/lib/gnulib/stdio.in.h
Copying file grub-core/lib/gnulib/stdlib.in.h
Copying file grub-core/lib/gnulib/strcasecmp.c
Copying file grub-core/lib/gnulib/strchrnul.c
Copying file grub-core/lib/gnulib/strchrnul.valgrind
Copying file grub-core/lib/gnulib/strdup.c
Copying file grub-core/lib/gnulib/streq.h
Copying file grub-core/lib/gnulib/strerror-override.c
Copying file grub-core/lib/gnulib/strerror-override.h
Copying file grub-core/lib/gnulib/strerror.c
Copying file grub-core/lib/gnulib/string.in.h
Copying file grub-core/lib/gnulib/strings.in.h
Copying file grub-core/lib/gnulib/stripslash.c
Copying file grub-core/lib/gnulib/strncasecmp.c
Copying file grub-core/lib/gnulib/strndup.c
Copying file grub-core/lib/gnulib/strnlen.c
Copying file grub-core/lib/gnulib/strnlen1.c
Copying file grub-core/lib/gnulib/strnlen1.h
Copying file grub-core/lib/gnulib/sys_stat.in.h
Copying file grub-core/lib/gnulib/sys_types.in.h
Copying file grub-core/lib/gnulib/sysexits.in.h
Copying file grub-core/lib/gnulib/time.in.h
Copying file grub-core/lib/gnulib/unistd--.h
Copying file grub-core/lib/gnulib/unistd-safer.h
Copying file grub-core/lib/gnulib/unistd.c
Copying file grub-core/lib/gnulib/unistd.in.h
Copying file grub-core/lib/gnulib/unitypes.in.h
Copying file grub-core/lib/gnulib/uniwidth.in.h
Copying file grub-core/lib/gnulib/uniwidth/cjk.h
Copying file grub-core/lib/gnulib/uniwidth/width.c
Copying file grub-core/lib/gnulib/vasnprintf.c
Copying file grub-core/lib/gnulib/vasnprintf.h
Copying file grub-core/lib/gnulib/verify.h
Copying file grub-core/lib/gnulib/vsnprintf.c
Copying file grub-core/lib/gnulib/warn-on-use.h
Copying file grub-core/lib/gnulib/wchar.in.h
Copying file grub-core/lib/gnulib/wcrtomb.c
Copying file grub-core/lib/gnulib/wctype-h.c
Copying file grub-core/lib/gnulib/wctype.in.h
Copying file grub-core/lib/gnulib/wcwidth.c
Copying file grub-core/lib/gnulib/xalloc-oversized.h
Copying file grub-core/lib/gnulib/xsize.c
Copying file grub-core/lib/gnulib/xsize.h
Copying file m4/00gnulib.m4
Copying file m4/__inline.m4
Copying file m4/absolute-header.m4
Copying file m4/alloca.m4
Copying file m4/argp.m4
Copying file m4/btowc.m4
Copying file m4/builtin-expect.m4
Copying file m4/chdir-long.m4
Copying file m4/close.m4
Replacing file m4/codeset.m4 (non-gnulib code backed up in m4/codeset.m4~) !!
Copying file m4/dirent_h.m4
Copying file m4/dirfd.m4
Copying file m4/dirname.m4
Copying file m4/double-slash-root.m4
Copying file m4/dup2.m4
Copying file m4/eealloc.m4
Copying file m4/errno_h.m4
Copying file m4/error.m4
Copying file m4/exponentd.m4
Copying file m4/extensions.m4
Replacing file m4/extern-inline.m4 (non-gnulib code backed up in m4/extern-inline.m4~) !!
Copying file m4/fchdir.m4
Replacing file m4/fcntl-o.m4 (non-gnulib code backed up in m4/fcntl-o.m4~) !!
Copying file m4/fcntl.m4
Copying file m4/fcntl_h.m4
Copying file m4/filenamecat.m4
Copying file m4/flexmember.m4
Copying file m4/float_h.m4
Copying file m4/fnmatch.m4
Copying file m4/fnmatch_h.m4
Copying file m4/fstat.m4
Copying file m4/getcwd.m4
Copying file m4/getdelim.m4
Copying file m4/getdtablesize.m4
Copying file m4/getline.m4
Copying file m4/getopt.m4
Copying file m4/getprogname.m4
Replacing file m4/glibc21.m4 (non-gnulib code backed up in m4/glibc21.m4~) !!
Copying file m4/gnulib-common.m4
Copying file m4/gnulib-tool.m4
Copying file m4/host-cpu-c-abi.m4
Copying file m4/include_next.m4
Copying file m4/intmax_t.m4
Replacing file m4/inttypes_h.m4 (non-gnulib code backed up in m4/inttypes_h.m4~) !!
Copying file m4/langinfo_h.m4
Copying file m4/largefile.m4
Replacing file m4/lib-ld.m4 (non-gnulib code backed up in m4/lib-ld.m4~) !!
Replacing file m4/lib-link.m4 (non-gnulib code backed up in m4/lib-link.m4~) !!
Replacing file m4/lib-prefix.m4 (non-gnulib code backed up in m4/lib-prefix.m4~) !!
Copying file m4/libunistring-base.m4
Copying file m4/limits-h.m4
Copying file m4/localcharset.m4
Copying file m4/locale-fr.m4
Copying file m4/locale-ja.m4
Copying file m4/locale-zh.m4
Copying file m4/locale_h.m4
Copying file m4/localeconv.m4
Replacing file m4/lock.m4 (non-gnulib code backed up in m4/lock.m4~) !!
Replacing file m4/longlong.m4 (non-gnulib code backed up in m4/longlong.m4~) !!
Copying file m4/lstat.m4
Copying file m4/malloc.m4
Copying file m4/malloca.m4
Copying file m4/math_h.m4
Copying file m4/mbrtowc.m4
Copying file m4/mbsinit.m4
Copying file m4/mbsrtowcs.m4
Copying file m4/mbstate_t.m4
Copying file m4/mbswidth.m4
Copying file m4/mbtowc.m4
Copying file m4/memchr.m4
Copying file m4/mempcpy.m4
Copying file m4/memrchr.m4
Copying file m4/mmap-anon.m4
Copying file m4/mode_t.m4
Copying file m4/msvc-inval.m4
Copying file m4/msvc-nothrow.m4
Copying file m4/multiarch.m4
Copying file m4/nl_langinfo.m4
Copying file m4/nocrash.m4
Copying file m4/off_t.m4
Copying file m4/open-cloexec.m4
Copying file m4/open.m4
Copying file m4/openat.m4
Copying file m4/pathmax.m4
Copying file m4/printf.m4
Copying file m4/pthread_rwlock_rdlock.m4
Copying file m4/rawmemchr.m4
Copying file m4/realloc.m4
Copying file m4/regex.m4
Copying file m4/save-cwd.m4
Replacing file m4/size_max.m4 (non-gnulib code backed up in m4/size_max.m4~) !!
Copying file m4/sleep.m4
Copying file m4/ssize_t.m4
Copying file m4/stat-time.m4
Copying file m4/stat.m4
Copying file m4/stdalign.m4
Copying file m4/stdbool.m4
Copying file m4/stddef_h.m4
Copying file m4/stdint.m4
Replacing file m4/stdint_h.m4 (non-gnulib code backed up in m4/stdint_h.m4~) !!
Copying file m4/stdio_h.m4
Copying file m4/stdlib_h.m4
Copying file m4/strcase.m4
Copying file m4/strchrnul.m4
Copying file m4/strdup.m4
Copying file m4/strerror.m4
Copying file m4/string_h.m4
Copying file m4/strings_h.m4
Copying file m4/strndup.m4
Copying file m4/strnlen.m4
Copying file m4/sys_socket_h.m4
Copying file m4/sys_stat_h.m4
Copying file m4/sys_types_h.m4
Copying file m4/sysexits.m4
Replacing file m4/threadlib.m4 (non-gnulib code backed up in m4/threadlib.m4~) !!
Copying file m4/time_h.m4
Copying file m4/unistd-safer.m4
Copying file m4/unistd_h.m4
Copying file m4/vasnprintf.m4
Copying file m4/vsnprintf.m4
Copying file m4/warn-on-use.m4
Copying file m4/wchar_h.m4
Replacing file m4/wchar_t.m4 (non-gnulib code backed up in m4/wchar_t.m4~) !!
Copying file m4/wcrtomb.m4
Copying file m4/wctype_h.m4
Copying file m4/wcwidth.m4
Replacing file m4/wint_t.m4 (non-gnulib code backed up in m4/wint_t.m4~) !!
Replacing file m4/xsize.m4 (non-gnulib code backed up in m4/xsize.m4~) !!
Creating grub-core/lib/gnulib/Makefile.am
Creating m4/gnulib-cache.m4
Creating m4/gnulib-comp.m4
Finished.

You may need to add #include directives for the following .h files.
  #include <fnmatch.h>
  #include <regex.h>
  #include <stdio.h>
  #include <stdlib.h>
  #include "argp.h"
  #include "error.h"
  #include "gettext.h"
  #include "mbswidth.h"
  #include "progname.h"
  #include "save-cwd.h"

You may need to use the following Makefile variables when linking.
Use them in <program>_LDADD when linking a program, or
in <library>_a_LDFLAGS or <library>_la_LDFLAGS when linking a library.
  $(LTLIBINTL) when linking with libtool, $(LIBINTL) otherwise
  $(LTLIBTHREAD) when linking with libtool, $(LIBTHREAD) otherwise

Don't forget to
  - add "grub-core/lib/gnulib/Makefile" to AC_CONFIG_FILES in ./configure.ac,
  - mention "gnulib" in SUBDIRS in grub-core/lib/Makefile.am,
  - mention "-I m4" in ACLOCAL_AMFLAGS in Makefile.am,
  - mention "m4/gnulib-cache.m4" in EXTRA_DIST in Makefile.am,
  - invoke gl_EARLY in ./configure.ac, right after AC_PROG_CC,
  - invoke gl_INIT in ./configure.ac.
patching file argp-parse.c
patching file argp-fmtstream.c
patching file argp-help.c
patching file mbswidth.c
patching file mbswidth.h
patching file regcomp.c
find: warning: ‘-iname’ matches against basenames only, but the given pattern contains a directory separator (‘/’), thus the expression will evaluate to false all the time.  Did you mean ‘-iwholename’?
Importing unicode...
./autogen.sh: line 20: python: command not found
running: AUTOPOINT=true LIBTOOLIZE=true autoreconf --verbose --install --force -I m4  --no-recursive
autoreconf: Entering directory `.'
autoreconf: running: true --force
autoreconf: running: aclocal -I m4 --force -I m4
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: running: /usr/bin/autoconf --include=m4 --force
autoreconf: running: /usr/bin/autoheader --include=m4 --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:327: installing 'build-aux/compile'
configure.ac:42: installing 'build-aux/config.guess'
configure.ac:42: installing 'build-aux/config.sub'
configure.ac:48: installing 'build-aux/install-sh'
configure.ac:48: installing 'build-aux/missing'
automake: error: cannot open < Makefile.util.am: No such file or directory
autoreconf: automake failed with exit status: 1
./bootstrap: autoreconf failed
find: warning: ‘-iname’ matches against basenames only, but the given pattern contains a directory separator (‘/’), thus the expression will evaluate to false all the time.  Did you mean ‘-iwholename’?
Importing unicode...
./autogen.sh: line 20: python: command not found
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-none
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for cmp... cmp
checking for bison... bison
checking for ranlib... ranlib
checking for gawk... (cached) gawk
checking whether make supports the include directive... yes (GNU style)
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... none
checking for flex... flex
checking lex output file root... lex.yy
checking lex library... none needed
checking whether yytext is a pointer... no
checking for bison... (cached) bison
checking whether make sets $(MAKE)... (cached) yes
checking whether ln -s works... yes
checking for makeinfo... no
checking for true... /bin/true
checking for gcc... (cached) gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to accept ISO C89... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking dependency style of gcc... (cached) none
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking for Minix Amsterdam compiler... no
checking for ar... ar
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... none
checking dependency style of gcc... none
checking for a Python interpreter with version >= 2.6... python2
checking for python2... /usr/bin/python2
checking for python2 version... 2.7
checking for python2 platform... linux2
checking for python2 script directory... ${prefix}/lib/python2.7/dist-packages
checking for python2 extension module directory... ${exec_prefix}/lib/python2.7/dist-packages
checking for g++... yes
checking for a sed that does not truncate output... /bin/sed
checking whether NLS is requested... yes
checking for msgfmt... /usr/bin/msgfmt
checking for gmsgfmt... /usr/bin/msgfmt
checking for xgettext... /usr/bin/xgettext
checking for msgmerge... /usr/bin/msgmerge
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for shared library run path origin... done
checking 32-bit host C ABI... no
checking for the common suffixes of directories in the library search path... lib,lib
checking for CFPreferencesCopyAppValue... no
checking for CFLocaleCopyCurrent... no
checking for GNU gettext in libc... yes
checking whether to use NLS... yes
checking where the gettext function comes from... libc
checking for special C compiler options needed for large files... (cached) no
checking for _FILE_OFFSET_BITS value needed for large files... (cached) no
checking whether byte ordering is bigendian... no
checking size of void *... 8
checking size of long... 8
checking size of off_t... 8
checking for help2man... no
checking for posix_memalign... yes
checking for memalign... yes
checking for getextmntent... no
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking sys/mount.h usability... yes
checking sys/mount.h presence... yes
checking for sys/mount.h... yes
checking sys/mnttab.h usability... no
checking sys/mnttab.h presence... no
checking for sys/mnttab.h... no
checking limits.h usability... yes
checking limits.h presence... yes
checking for limits.h... yes
checking for sys/types.h... (cached) yes
checking for features.h... yes
checking for linewrap.h... no
checking for unistd.h... (cached) yes
checking for sys/param.h... (cached) yes
checking for sys/socket.h... yes
checking for dirent.h... yes
checking for fnmatch.h... yes
checking for wctype.h... yes
checking for sys/stat.h... (cached) yes
checking for getopt.h... yes
checking for sys/cdefs.h... yes
checking for langinfo.h... yes
checking for limits.h... (cached) yes
checking for xlocale.h... no
checking for wchar.h... yes
checking for sys/mman.h... yes
checking for malloc.h... yes
checking for sys/time.h... yes
checking for stdint.h... (cached) yes
checking for strings.h... (cached) yes
checking for sysexits.h... yes
checking for crtdefs.h... no
checking sys/mkdev.h usability... no
checking sys/mkdev.h presence... no
checking for sys/mkdev.h... no
checking sys/sysmacros.h usability... yes
checking sys/sysmacros.h presence... yes
checking for sys/sysmacros.h... yes
checking for struct statfs.f_fstypename... no
checking for struct statfs.f_mntfromname... no
checking util.h usability... no
checking util.h presence... no
checking for util.h... no
checking whether -Wtrampolines work... yes
checking for gcc... gcc
checking for size_t... yes
checking for working alloca.h... yes
checking for alloca... yes
checking for inline... inline
checking for C/C++ restrict keyword... __restrict
checking whether clearerr_unlocked is declared... yes
checking whether feof_unlocked is declared... yes
checking whether ferror_unlocked is declared... yes
checking whether fflush_unlocked is declared... yes
checking whether fgets_unlocked is declared... yes
checking whether fputc_unlocked is declared... yes
checking whether fputs_unlocked is declared... yes
checking whether fread_unlocked is declared... yes
checking whether fwrite_unlocked is declared... yes
checking whether getc_unlocked is declared... yes
checking whether getchar_unlocked is declared... yes
checking whether putc_unlocked is declared... yes
checking whether putchar_unlocked is declared... yes
checking for flockfile... yes
checking for funlockfile... yes
checking for btowc... yes
checking for _set_invalid_parameter_handler... no
checking for fchdir... yes
checking for fcntl... yes
checking for symlink... yes
checking for mempcpy... yes
checking for fnmatch... yes
checking for isblank... yes
checking for iswctype... yes
checking for mbsrtowcs... yes
checking for wmemchr... yes
checking for wmemcpy... yes
checking for wmempcpy... yes
checking for getdelim... yes
checking for getdtablesize... yes
checking for getprogname... no
checking for getexecname... no
checking for mbsinit... yes
checking for mbrtowc... yes
checking for isascii... yes
checking for mprotect... yes
checking for nl_langinfo... yes
checking for lstat... yes
checking for openat... yes
checking for sleep... yes
checking for strdup... yes
checking for strndup... yes
checking for pipe... yes
checking for vasnprintf... no
checking for snprintf... yes
checking for wcrtomb... yes
checking for iswcntrl... yes
checking for wcwidth... yes
checking whether strerror_r is declared... yes
checking for strerror_r... yes
checking whether strerror_r returns char *... yes
checking whether <wchar.h> uses 'inline' correctly... yes
checking for nl_langinfo and CODESET... yes
checking for a traditional french locale... none
checking whether the preprocessor supports include_next... yes
checking whether system header files limit the line length... no
checking whether // is distinct from /... no
checking for complete errno.h... yes
checking whether fchdir is declared... yes
checking for working fcntl.h... yes
checking for pid_t... yes
checking for mode_t... yes
checking for mbstate_t... yes
checking whether stat file-mode macros are broken... no
checking for nlink_t... yes
checking whether getcwd (NULL, 0) allocates memory for result... yes
checking for getcwd with POSIX signature... yes
checking whether getdelim is declared... yes
checking whether getdtablesize is declared... yes
checking whether getline is declared... yes
checking for getopt.h... (cached) yes
checking for getopt_long_only... yes
checking whether getopt is POSIX compatible... yes
checking for working GNU getopt function... yes
checking for working GNU getopt_long function... yes
checking whether limits.h has LLONG_MAX, WORD_BIT, ULLONG_WIDTH etc.... yes
checking for wchar_t... yes
checking for good max_align_t... yes
checking whether NULL can be used in arbitrary expressions... yes
checking whether imported symbols can be declared weak... yes
checking whether the linker supports --as-needed... yes
checking whether the linker supports --push-state... yes
checking pthread.h usability... yes
checking pthread.h presence... yes
checking for pthread.h... yes
checking for multithread API to use... posix
checking whether malloc, realloc, calloc are POSIX compliant... yes
checking for stdlib.h... (cached) yes
checking for GNU libc compatible malloc... yes
checking for unsigned long long int... yes
checking for long long int... yes
checking for a traditional japanese locale... none
checking for a transitional chinese locale... none
checking for a french Unicode locale... none
checking for mmap... yes
checking for MAP_ANONYMOUS... yes
checking whether memchr works... yes
checking whether memrchr is declared... yes
checking for O_CLOEXEC... yes
checking for promoted mode_t type... mode_t
checking whether lstat correctly handles trailing slash... yes
checking whether alarm is declared... yes
checking whether we are using the GNU C Library >= 2.1 or uClibc... yes
checking for stdbool.h that conforms to C99... yes
checking for _Bool... yes
checking for wint_t... yes
checking whether wint_t is too small... no
checking whether stdint.h conforms to C99... yes
checking whether stdint.h predates C++11... no
checking whether stdint.h has UINTMAX_WIDTH etc.... yes
checking whether strdup is declared... yes
checking whether strerror(0) succeeds... yes
checking whether strndup is declared... yes
checking whether strnlen is declared... yes
checking for struct timespec in <time.h>... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for intmax_t... yes
checking where to find the exponent in a 'double'... word 1 bit 20
checking whether snprintf returns a byte count as in C99... yes
checking for snprintf... (cached) yes
checking for strnlen... yes
checking for wcslen... yes
checking for wcsnlen... yes
checking for mbrtowc... (cached) yes
checking for wcrtomb... (cached) yes
checking whether _snprintf is declared... no
checking whether vsnprintf is declared... yes
checking for alloca as a compiler built-in... yes
checking whether program_invocation_name is declared... yes
checking whether program_invocation_short_name is declared... yes
checking whether program_invocation_name is defined... yes
checking whether program_invocation_short_name is defined... yes
checking whether btowc(0) is correct... yes
checking whether btowc(EOF) is correct... guessing yes
checking for __builtin_expect... yes
checking whether this system has an arbitrary file name length limit... yes
checking for dirfd... yes
checking whether dirfd is declared... yes
checking whether dirfd is a macro... no
checking whether // is distinct from /... (cached) no
checking whether dup2 works... yes
checking for error_at_line... yes
checking whether fcntl handles F_DUPFD correctly... yes
checking whether fcntl understands F_DUPFD_CLOEXEC... needs runtime check
checking for flexible array member... yes
checking whether conversion from 'int' to 'long double' works... yes
checking for working POSIX fnmatch... yes
checking for working getdelim function... yes
checking whether getdtablesize works... yes
checking for getline... yes
checking for working getline function... yes
checking whether program_invocation_name is declared... (cached) yes
checking whether program_invocation_short_name is declared... (cached) yes
checking whether __argv is declared... no
checking whether langinfo.h defines CODESET... yes
checking whether langinfo.h defines T_FMT_AMPM... yes
checking whether langinfo.h defines ALTMON_1... yes
checking whether langinfo.h defines ERA... yes
checking whether langinfo.h defines YESEXPR... yes
checking whether the compiler supports the __inline keyword... yes
checking whether locale.h conforms to POSIX:2001... yes
checking whether struct lconv is properly defined... yes
checking for pthread_rwlock_t... yes
checking whether pthread_rwlock_rdlock prefers a writer to a reader... no
checking for stdlib.h... (cached) yes
checking for GNU libc compatible malloc... (cached) yes
checking whether mbrtowc handles incomplete characters... guessing yes
checking whether mbrtowc works as well as mbtowc... guessing yes
checking whether mbrtowc handles a NULL pwc argument... guessing yes
checking whether mbrtowc handles a NULL string argument... guessing yes
checking whether mbrtowc has a correct return value... guessing yes
checking whether mbrtowc returns 0 when parsing a NUL character... guessing yes
checking whether mbrtowc works on empty input... yes
checking whether the C locale is free of encoding errors... no
checking whether mbrtowc handles incomplete characters... (cached) guessing yes
checking whether mbrtowc works as well as mbtowc... (cached) guessing yes
checking whether mbrtowc handles incomplete characters... (cached) guessing yes
checking whether mbrtowc works as well as mbtowc... (cached) guessing yes
checking whether mbsrtowcs works... guessing yes
checking whether mbswidth is declared in <wchar.h>... no
checking for mbstate_t... (cached) yes
checking for mempcpy... (cached) yes
checking for memrchr... yes
checking whether YESEXPR works... yes
checking whether open recognizes a trailing slash... yes
checking whether program_invocation_name is declared... (cached) yes
checking whether program_invocation_short_name is declared... (cached) yes
checking for rawmemchr... yes
checking for stdlib.h... (cached) yes
checking for GNU libc compatible realloc... yes
checking for working re_compile_pattern... no
checking libintl.h usability... yes
checking libintl.h presence... yes
checking for libintl.h... yes
checking whether isblank is declared... yes
checking for stdint.h... (cached) yes
checking for SIZE_MAX... yes
checking whether sleep is declared... yes
checking for working sleep... yes
checking for ssize_t... yes
checking whether stat handles trailing slashes on files... yes
checking for struct stat.st_atim.tv_nsec... yes
checking whether struct stat.st_atim is of type struct timespec... yes
checking for struct stat.st_birthtimespec.tv_nsec... no
checking for struct stat.st_birthtimensec... no
checking for struct stat.st_birthtim.tv_nsec... no
checking for working stdalign.h... yes
checking for good max_align_t... (cached) yes
checking whether NULL can be used in arbitrary expressions... (cached) yes
checking which flavor of printf attribute matches inttypes macros... system
checking for strcasecmp... yes
checking for strncasecmp... yes
checking whether strncasecmp is declared... yes
checking for strchrnul... yes
checking whether strchrnul works... yes
checking for working strerror function... yes
checking for working strndup... yes
checking for working strnlen... yes
checking for nlink_t... (cached) yes
checking for ptrdiff_t... yes
checking for vsnprintf... yes
checking whether snprintf respects a size of 1... yes
checking whether printf supports POSIX/XSI format strings with positions... yes
checking whether mbrtowc handles incomplete characters... (cached) guessing yes
checking whether mbrtowc works as well as mbtowc... (cached) guessing yes
checking whether wcrtomb return value is correct... guessing yes
checking whether iswcntrl works... yes
checking for towlower... yes
checking for wctype_t... yes
checking for wctrans_t... yes
checking whether wcwidth is declared... yes
checking whether wcwidth works reasonably in UTF-8 locales... yes
checking for stdint.h... (cached) yes
checking which extra warnings work...  -Wextra -Wattributes -Wendif-labels -Winit-self -Wint-to-pointer-cast -Winvalid-pch -Wmissing-field-initializers -Wnonnull -Woverflow -Wvla -Wpointer-to-int-cast -Wstrict-aliasing -Wvariadic-macros -Wvolatile-register-var -Wpointer-sign -Wmissing-include-dirs -Wmissing-prototypes -Wmissing-declarations -Wformat=2
checking for x86_64-gcc... no
checking for x86_64-egcs... no
checking for x86_64-cc... no
checking for gcc... gcc
checking for x86_64-objcopy... no
checking for objcopy... objcopy
checking for x86_64-strip... no
checking for strip... strip
checking for x86_64-nm... no
checking for nm... nm
checking for x86_64-ranlib... no
checking for ranlib... ranlib
checking which extra warnings work...  -Wextra -Wattributes -Wendif-labels -Winit-self -Wint-to-pointer-cast -Winvalid-pch -Wmissing-field-initializers -Wnonnull -Woverflow -Wvla -Wpointer-to-int-cast -Wstrict-aliasing -Wvariadic-macros -Wvolatile-register-var -Wpointer-sign -Wmissing-include-dirs -Wmissing-prototypes -Wmissing-declarations -Wformat=2
checking if compiling with clang... no
checking for options to compile assembly... 
checking whether -freg-struct-return works... yes
checking for options to get soft-float... -msoft-float
checking whether -fno-dwarf2-cfi-asm works... yes
checking whether -mno-stack-arg-probe works... yes
checking whether -fno-asynchronous-unwind-tables works... yes
checking whether -fno-unwind-tables works... yes
checking for target linking format... -melf_x86_64
checking whether option -mcmodel=large works... yes
checking whether option -mno-red-zone works... yes
checking whether option -Qn works... no
checking whether `gcc' has `-fPIE' as default... yes
checking whether linker accepts -no-pie... yes
checking whether linker accepts -nopie... no
checking whether linker needs disabling of PIE to work... no
checking whether `gcc' has `-fPIC' as default... no
checking whether `gcc' accepts `-fstack-protector'... yes
checking whether `gcc' accepts `-mstack-arg-probe'... no
checking if C symbols get an underscore after compilation... no
checking whether target compiler is working... yes
checking whether objcopy works for absolute addresses... yes
checking whether linker accepts --build-id=none... yes
checking whether nm works... yes
checking whether nm accepts -P... yes
checking whether nm accepts --defined-only... yes
checking whether -nostdinc -isystem works... yes
checking whether -Wtrampolines work... yes
checking for FREETYPE... yes
checking ft2build.h usability... yes
checking ft2build.h presence... yes
checking for ft2build.h... yes
checking whether byte ordering is bigendian... no
checking for BUILD_FREETYPE... yes
checking ft2build.h usability... yes
checking ft2build.h presence... yes
checking for ft2build.h... yes
checking for fuse_main_real in -lfuse... no
checking libdevmapper.h usability... no
checking libdevmapper.h presence... no
checking for libdevmapper.h... no
checking for lzma_code in -llzma... yes
checking lzma.h usability... yes
checking lzma.h presence... yes
checking for lzma.h... yes
checking libzfs.h usability... no
checking libzfs.h presence... no
checking for libzfs.h... no
checking libnvpair.h usability... no
checking libnvpair.h presence... no
checking for libnvpair.h... no
checking for libzfs_init in -lzfs... no
checking whether ln -s can handle directories properly... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: error: cannot find input file: `Makefile.in'
make: *** No targets specified and no makefile found.  Stop.
./boards/x86/common/scripts/build_grub.sh: line 39: ./grub-mkimage: No such file or directory
~/projects/zephyr/zephyr/boards/x86/common/scripts/grub ~/projects/zephyr/zephyr
cp: cannot stat 'src/grub_x86_64.efi': No such file or directory
```

A few spots look suspicious:

```
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:327: installing 'build-aux/compile'
configure.ac:42: installing 'build-aux/config.guess'
configure.ac:42: installing 'build-aux/config.sub'
configure.ac:48: installing 'build-aux/install-sh'
configure.ac:48: installing 'build-aux/missing'
automake: error: cannot open < Makefile.util.am: No such file or directory
autoreconf: automake failed with exit status: 1
```

```
Importing unicode...
./autogen.sh: line 20: python: command not found
```

```
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: error: cannot find input file: `Makefile.in'
```

--- a/drivers/usb/device/usb_dc_nrfx.c
+++ b/drivers/usb/device/usb_dc_nrfx.c
@@ -847,7 +847,7 @@ static inline void usbd_work_process_setup(struct nrf_usbd_ep_ctx *ep_ctx)
 	usbd_setup->wLength = nrf_usbd_setup_wlength_get(NRF_USBD);
 	ep_ctx->buf.len = sizeof(struct usb_setup_packet);
 
-	LOG_DBG("SETUP: r:%d rt:%d v:%d i:%d l:%d",
+	LOG_DBG("SETUP: bR:0x%02x bmRT:0x%02x wV:0x%04x wI:0x%04x wL:%d",
 		(u32_t)usbd_setup->bRequest,
 		(u32_t)usbd_setup->bmRequestType,
 		(u32_t)usbd_setup->wValue,
@@ -862,9 +862,8 @@ static inline void usbd_work_process_setup(struct nrf_usbd_ep_ctx *ep_ctx)
 	if ((REQTYPE_GET_DIR(usbd_setup->bmRequestType)
 	     == REQTYPE_DIR_TO_DEVICE)
 	    && (usbd_setup->wLength)) {
-		struct nrf_usbd_ctx *ctx = get_usbd_ctx();
-
-		ctx->ctrl_read_len -= usbd_setup->wLength;
+		ctx->ctrl_read_len = usbd_setup->wLength;
+		/* Allow data chunk on EP0 OUT */
 		nrfx_usbd_setup_data_clear();
 	} else {
 		ctx->ctrl_read_len = 0U;
@@ -1015,6 +1014,7 @@ static void usbd_event_transfer_ctrl(nrfx_usbd_evt_t const *const p_event)
 
 			if (ctx->ctrl_read_len > ep_ctx->buf.len) {
 				ctx->ctrl_read_len -= ep_ctx->buf.len;
+				/* Allow next data chunk on EP0 OUT */
 				nrfx_usbd_setup_data_clear();
 			} else {
 				ctx->ctrl_read_len = 0U;

